WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewViewModel.swift

```diff
@@ -187,22 +253,81 @@ final class ProductDetailPreviewViewModel: ObservableObject {
     }
 }
 
+// MARK: - Undo edits
+//
+extension ProductDetailPreviewViewModel {
+    func undoEdits(in field: ProductDetailField) {
+        guard let generatedAIProduct else {
+            return
+        }
+
+        guard let option = options[safe: selectedOptionIndex] else {
+            return
+        }
```

Comment:
I think `option` is unused here.

Reply:
Good catch. Removed in d2a51b7a25e5e11c4936f4dacc3289eef1423aad.